src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -211,9 +211,9 @@ public void GetIssues_NoIssuesOnServer_ReturnsEmptyList()
         public void GetSuppressedIssues_WhenProjectHasModulesAndIssueIsModuleLevelFound_ReturnsExpectedIssue()
         {
             // Arrange
-            var sonarQubeIssue1 = new SonarQubeIssue(null, null, null, "message", "sqkey:sqkey:projectId2", "S1", true, flows: null);
-            var sonarQubeIssue2 = new SonarQubeIssue(null, null, null, "message", "sqkey:sqkey:projectId", "S2", true, flows: null);
-            var sonarQubeIssue3 = new SonarQubeIssue("foo\\bar.cs", "hash", 12, "message", "sqkey:sqkey:projectId", "S3", true, flows: null);
+            var sonarQubeIssue1 = new SonarQubeIssue(null, null, "message", "sqkey:sqkey:projectId2", "S1", true, SonarQubeIssueSeverity.Blocker, null, flows: null);
```

Comment:
The "null" before the message was the line number.
I now instead set the text range field, which is after the severity.